### PR TITLE
Libraries and clearing display

### PR DESCRIPTION
**Pull request by Suocesky**
With the repository merge, some extra libraries were needed and installed. 
Instances of = Tank('', '') have been removed and been replaced with either a clearTank function that sets the current tank to empty, or a bool function checking if the current tank is empty or not.
The display and tank name header on the web page will now clear themselves when the current tank is deleted from the tank list.

### CODE REVIEW

File: extras/webClient/lib/components/app_drawer.dart [hunk 1]
```diff
@@ -76,12 +76,13 @@ class AppDrawer extends StatelessWidget {
           Align(
             alignment: Alignment.topRight,
             child: FloatingActionButton(
-                onPressed: () {
-                  appData.removeTank(appData.currentTank);
-                  appData.currentTank = Tank("", "");
-                },
-                tooltip: 'Remove Tank',
-                child: const Icon(Icons.delete_sharp)),
+              onPressed: () {
```
Comment (jgfoster) [inline extras/webClient/lib/components/app_drawer.dart:79]: Is this proper format?

File: extras/webClient/lib/model/app_data.dart [hunk 2]
```diff
@@ -78,16 +84,25 @@ class AppData with ChangeNotifier {
     // ignore result
     await TcInterface.instance.get(tank.ip, 'current');
     _tankList.add(tank);
+    refreshDisplay();
     notifyListeners();
     writeTankList(tankList);
   }
 
   void removeTank(tank) {
     _tankList.remove(tank);
```
Comment (jgfoster) [inline extras/webClient/lib/model/app_data.dart:93]: Only clear and refresh if current tank is displayed.

**Review by jgfoster — CHANGES_REQUESTED** [review]

File: extras/webClient/lib/components/app_drawer.dart (hunk 1 above)
Comment (jgfoster) [reply extras/webClient/lib/components/app_drawer.dart:79]: Looks good!

File: extras/webClient/lib/model/app_data.dart (hunk 2 above)
Comment (jgfoster) [reply extras/webClient/lib/model/app_data.dart:93]: Looks good!

**Review by jgfoster — APPROVED** [review]

**Review by prestoncarman — APPROVED** [review]

**Pull request merged**